air-quality-ui/system_tests/ui/summary_page.spec.ts

```diff
@@ -1,78 +1,570 @@
 
-  test('Verify that a city with no in-situ data still show on grid', async ({
-    vairifySummaryPage,
-  }) => {
-    const count = await vairifySummaryPage.textCellSearch('Kyiv')
-    expect(count).toEqual(1)
-  })
+    test('Verify on page load the forecast API is called once', async ({
+      summaryPage,
+      page,
+      basePage,
+    }) => {
+      const requestArray = await summaryPage.captureNetworkRequestsAsArray(
+        page,
+        'GET',
+        basePage.baseAPIURL + forecastAPIEndpoint,
+      )
+      await page.clock.setFixedTime(systemTime)
+      await summaryPage.goTo()
+      expect(requestArray.length).toEqual(1)
+    })
+
+    test('Verify on page load the measurement summary API calls have correct params', async ({
+      summaryPage,
+      page,
+      basePage,
+    }) => {
+      const requestArray = await summaryPage.captureNetworkRequestsAsArray(
+        page,
+        'GET',
+        basePage.baseAPIURL + measurementSummaryAPIEndpoint,
+      )
+      await page.clock.setFixedTime(systemTime)
+      await summaryPage.goTo()
+      const expectedMeasurementBaseTimeArray = [
+        '2024-06-09T00%3A00%3A00.000Z',
+        '2024-06-09T03%3A00%3A00.000Z',
+        '2024-06-09T06%3A00%3A00.000Z',
+        '2024-06-09T09%3A00%3A00.000Z',
+        '2024-06-09T12%3A00%3A00.000Z',
+        '2024-06-09T15%3A00%3A00.000Z',
+        '2024-06-09T18%3A00%3A00.000Z',
+        '2024-06-09T21%3A00%3A00.000Z',
+        '2024-06-10T00%3A00%3A00.000Z',
+        '2024-06-10T03%3A00%3A00.000Z',
+        '2024-06-10T06%3A00%3A00.000Z',
+        '2024-06-10T09%3A00%3A00.000Z',
+        '2024-06-10T12%3A00%3A00.000Z',
+        '2024-06-10T15%3A00%3A00.000Z',
+        '2024-06-10T18%3A00%3A00.000Z',
+      ]
+      for (const request in requestArray) {
+        expect(requestArray[request]).toContain(
+          `measurement_base_time=${expectedMeasurementBaseTimeArray[request]}&measurement_time_range=90&location_type=city`,
+        )
+      }
+    })
 
-  test('Check data is displayed correctly on grid', async ({
-    vairifySummaryPage,
-  }) => {
-    // first 6 values for Kampala, Abu Dhabi, Zurich and Kyiv respectively
-    const expectedData = [
-      ['2', '6', '4', '16.1', '76', '19 Jun 09:00'],
-      ['4', '5', '1', '30.3', '52.8', '19 Jun 12:00'],
-      ['2', '1', '1', '17.2', '15.8', '19 Jun 12:00'],
-      ['2', '', '', '7', '', '24 Jun 09:00'],
-    ]
-
-    await vairifySummaryPage.assertGridValues(expectedData)
+    test('Verify on page load the measurement summary API is called proportionately', async ({
+      summaryPage,
+      page,
+      basePage,
+    }) => {
+      const requestArray = await summaryPage.captureNetworkRequestsAsArray(
+        page,
+        'GET',
+        basePage.baseAPIURL + measurementSummaryAPIEndpoint,
+      )
+      await page.clock.setFixedTime(systemTime)
+      await summaryPage.goTo()
+      const expectedNumberofRequests =
```

Comment:
Do we gain much from calculating this? We haven't calculated the base forecast time anywhere else so maybe the test can either just know the number, or can pass the base forecast time in?
This is the only place we're using the function to calculate the base forecast time, so if we update we can remove the function from the page I think.